K2Bridge/Factories/BucketFactory.cs

```diff
@@ -26,7 +27,7 @@ public static DateHistogramBucket CreateDateHistogramBucketFromDataRow(DataRow r
             // TODO: timestamp is always the first row (probably named _2), and count will be named _count
             // we currently mix index and column names, need to check if we can enhance this logic
             // workitem 15050
-            var timestamp = row[(int)BucketColumnNames.Timestamp];
+            var timestamp = row[(int)BucketColumnNames.SummarizeByColumn];
```

Comment:
is it still 'timestamp' or maybe
var summarize 
?

Reply:
👍🏻

Reply:
Why was it resolved? it's still `var timestamp`

Reply:
We discussed offline with Elad; in the case of a DateHistogramBucket, the "summarize by" column is a timestamp indeed, so the name of the variable is appropriate.